mani_skill/assets/robots/so101/so101.urdf

```diff
@@ -337,11 +337,11 @@
 
   <!-- Joint from gripper to moving_jaw_so101_v1 -->
   <joint name="gripper" type="revolute">
-    <origin xyz="0.0202 0.0188 -0.0234" rpy="1.5708 -0.909 -1.41553e-15"/>
+    <origin xyz="0.0202 0.0188 -0.0234" rpy="1.5708 -5.24284e-08 -1.41553e-15"/>
     <parent link="gripper_link"/>
     <child link="moving_jaw_so101_v1_link"/>
     <axis xyz="0 0 1"/>
-    <limit effort="0" velocity="10" lower="-1.13778" upper="1.16081"/>
+    <limit effort="10" velocity="10" lower="-0.174533" upper="2.0944"/>
```

Comment:
we don't use effort but just checking these values of 10 are copied from the original so101 urdf, the new calibrated one here right https://github.com/TheRobotStudio/SO-ARM100/blob/main/Simulation/SO101/so101_new_calib.urdf